index.js

```diff
@@ -0,0 +1,144 @@
+const header = document.createElement('header');
+
+const headerh1 = document.createElement('h1');
+headerh1.setAttribute('id', 'logo');
+headerh1.innerText = 'Techno Mobile';
+
+const button = document.createElement('button');
+button.setAttribute('type', 'button');
```

Comment:
- [ ] You should have an event listener for opening the menu.

I can see that you created a button for the mobile menu but it doesn't have any event listeners to open the mobile menu.